package.json

```diff
@@ -116,7 +111,12 @@
   "devDependencies": {
     "@codeceptjs/allure-legacy": "1.0.2",
     "@playwright/test": "1.50.1",
+    "@types/express": "4.17.21",
     "@types/jest": "29.5.14",
+    "@types/multer": "1.4.7",
+    "@types/serve-favicon": "2.5.5",
+    "@types/session-file-store": "1.2.0",
+    "@types/supertest": "^6.0.2",
```

Comment:
multer, serve-favicon and session-file-store all look to have been reverted to a previous version - is that intentional?